programs/server/config.yaml

```diff
@@ -145,7 +145,10 @@ max_connections: 4096
 keep_alive_timeout: 3
 
 # Enable telemetry. This is used to collect the version and runtime environment information to Timeplus, Inc.
-telemetry_enabled: true
+telemetry_enabled:
+    "@from_env": PROTON_TELEMETRY_ENABLED
```

Comment:
Suggest keep using `TELEMETRY_ENABLED` Otherwise if the someone set Docker env or system env, they have to change the env name. Assume there is no name conflicting with other products.